lambdas/data-migration2/tests/test-pdrs.js

```diff
@@ -0,0 +1,336 @@
+  createdAt: Date.now(),
+  updatedAt: Date.now(),
+});
+
+let collectionsModel;
+let providersModel;
+let pdrsModel;
+
+const testDbName = `data_migration_2_${cryptoRandomString({ length: 10 })}`;
+const testDbUser = 'postgres';
+
+test.before(async (t) => {
+  process.env.stackName = cryptoRandomString({ length: 10 });
+  process.env.system_bucket = cryptoRandomString({ length: 10 });
+
+  process.env.CollectionsTable = cryptoRandomString({ length: 10 });
+  process.env.ProvidersTable = cryptoRandomString({ length: 10 });
+  process.env.PdrsTable = cryptoRandomString({ length: 10 });
+
+  await createBucket(process.env.system_bucket);
+
+  collectionsModel = new Collection();
+  await collectionsModel.createTable();
+
+  providersModel = new Provider();
+  await providersModel.createTable();
+
+  pdrsModel = new Pdr();
+  await pdrsModel.createTable();
+
+  t.context.knexAdmin = await getKnexClient({
+    env: {
+      ...localStackConnectionEnv,
+      migrationDir,
+    },
+  });
+  await t.context.knexAdmin.raw(`create database "${testDbName}";`);
+  await t.context.knexAdmin.raw(`grant all privileges on database "${testDbName}" to "${testDbUser}"`);
+
+  t.context.knex = await getKnexClient({
+    env: {
+      ...localStackConnectionEnv,
+      PG_DATABASE: testDbName,
+      migrationDir,
+    },
+  });
+
+  await t.context.knex.migrate.latest();
+});
+
+test.beforeEach(async (t) => {
+  const testCollection = fakeCollectionFactory();
+  const testProvider = fakeProviderFactory({
+    encrypted: true,
+    privateKey: 'key',
+    cmKeyId: 'key-id',
+    certificateUri: 'uri',
+  });
+
+  t.context.testCollection = testCollection;
+  t.context.testProvider = testProvider;
+
+  t.context.collectionCumulusId = await migrateFakeCollectionRecord(testCollection, t.context.knex);
+  t.context.providerCumulusId = await migrateFakeProviderRecord(testProvider, t.context.knex);
+});
+
+test.afterEach.always(async (t) => {
+  await t.context.knex(tableNames.pdrs).del();
+  await t.context.knex(tableNames.providers).del();
+  await t.context.knex(tableNames.collections).del();
+});
+
+test.after.always(async (t) => {
+  await recursivelyDeleteS3Bucket(process.env.system_bucket);
+
+  await providersModel.deleteTable();
+  await collectionsModel.deleteTable();
+  await pdrsModel.deleteTable();
+
+  await t.context.knex.destroy();
+  await t.context.knexAdmin.raw(`drop database if exists "${testDbName}"`);
+  await t.context.knexAdmin.destroy();
+});
+
+test.serial('migratePdrRecord correctly migrates PDR record', async (t) => {
+  const { knex, testCollection, testProvider, collectionCumulusId, providerCumulusId } = t.context;
+
+  const testPdr = generateTestPdr(testCollection.name, testProvider.id);
+  await migratePdrRecord(testPdr, knex);
+
+  const record = await t.context.knex.queryBuilder()
+    .select()
+    .table(tableNames.pdrs)
+    .where({ name: testPdr.pdrName })
+    .first();
+
+  t.like(
+    omit(record, ['cumulus_id']),
```

Comment:
If you use `t.like` for comparison in tests, then you don't need to do an `omit` of properties that exist on the first argument but not the second. That's the whole beauty of `t.like`: it ignores properties that are "extra" and only compares the properties that exist from the second argument on the first:
```suggestion
    record, 
```

https://github.com/avajs/ava/blob/master/docs/03-assertions.md#likevalue-selector-message

Reply:
OH, that makes sense a lot of sense